addons/web/static/src/js/views/control_panel/control_panel_model.js

```diff
@@ -14,6 +14,16 @@ var _t = core._t;
 var DEFAULT_TIMERANGE = controlPanelViewParameters.DEFAULT_TIMERANGE;
 var TIME_RANGE_OPTIONS = controlPanelViewParameters.TIME_RANGE_OPTIONS;
 var COMPARISON_TIME_RANGE_OPTIONS = controlPanelViewParameters.COMPARISON_TIME_RANGE_OPTIONS;
+var OPTION_GENERATORS = controlPanelViewParameters.OPTION_GENERATORS;
+
+// returns a predicate used to test if two arrays have the same basic content
+const isIso = (id1) => {
```

Comment:
I would probably rename this function because isSomething should be or return a bool, not a predicate

Reply:
IsEqualTo should be ok I think